python-avd/pyavd/_eos_cli_config_gen/schema/schema_fragments/router_isis.schema.yml

```diff
@@ -393,3 +368,30 @@ keys:
       eos_cli:
         type: str
         description: Multiline EOS CLI rendered directly on the router isis in the final EOS configuration.
+      address_family:
+        type: list
+        items:
```

Comment:
Please remove the items key and everything below, since it will only take up space in the doc table.

Reply:
Fixed!